yarn.lock

```diff
@@ -1755,14 +1755,14 @@
     npmlog "^4.1.2"
```

Comment:
chyba tego pliku nie commitujemy? czy się mylę Panowie?

Reply:
hmm nie jest w .gitignore

Reply:
Commitujemy, bo yarn.lock trzyma znacznie bardziej szczegolowe informacje na temat zainstalowanych paczek z package.json i jest to potrzebne do tego, zeby na kazdej maszynie instalowaly sie dokladnie te same paczki.